ramls/requests-report.raml

```diff
@@ -0,0 +1,34 @@
+#%RAML 1.0
+title: Circulation
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Circulation Business Logic API
+    content: <b>API for report generation</b>
+
+types:
+  requests: !include requests.json
+  error: !include raml-util/schemas/error.schema
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  language: !include raml-util/traits/language.raml
+  pageable: !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  validate: !include raml-util/traits/validation.raml
```

Comment:
Are any of these traits honored in the code? I don't see a way to page the results, we return the entire json array. I don't think searching makes sense on an ID specific endpoint like this. I don't see how language would be used here. There isn't a body so there really isn't anything to validate. If we do expect to use any of these traits, we should use the `is: [trait]` syntax to indicate the code cares about the trait and then have some code that acts on the query parameters.

Reply:
Updated. None of these traits are respected in the code.

Reply:
`collection-get` requires `language` trait

Reply:
Yeah, I saw that. Oh well, I guess we'll just ignore the `lang` query argument like almost every other module does. 😄